### PR TITLE
Added ids to pod status entires.

### PR DESCRIPTION
**Pull request by ljelinkova**
I need to add some ids to the Deployments page for E2E testing. I've tried to do it myself but I am not sure if it works correctly and/or it is done right. So, any inputs would be appreciated.

### CODE REVIEW

**Comment by joshuawilson** [comment]
[test]

File: src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <ng-template #elsePods>
       <div>
         Pod status:
-        <li *ngFor="let column of pods.pods">
+        <li *ngFor="let column of pods.pods"  [attr.id]="'pod_status_' + column[0]">
```
Comment (joshuawilson) [inline src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html]: What does `[attr.id]` do? Why not just use `id`?

Comment (ljelinkova) [reply src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html]: I've used this answer in StackOverflow 

https://stackoverflow.com/questions/34568497/how-to-set-dynamic-idngfor-in-ionic2-angular2

I've just tried id = "'pod_status_' + column[0]" and it did not work.

Comment (joshuawilson) [reply src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html]: good to know, thanks

**Pull request merged**